ngi_reports/utils/entities.py

```diff
@@ -491,169 +610,41 @@ def populate(self, log, organism_names, **kwargs):
         for key in self.accredited:
             self.accredited[key] = proj_details.get(f"accredited_({key})")
 
-        if "hiseqx" in proj_details.get("sequencing_platform", ""):
-            self.is_hiseqx = True
-
         self.sequencing_setup = proj_details.get("sequencing_setup")
 
-        for sample_id, sample in sorted(proj.get("samples", {}).items()):
+        for sample_id, sample_info in sorted(proj.get("samples", {}).items()):
             if kwargs.get("samples", []) and sample_id not in kwargs.get("samples", []):
                 log.info(
                     f"Will not include sample {sample_id} as it is not in given list"
                 )
                 continue
 
-            customer_name = sample.get("customer_name", "NA")
-            # Get once for a project
-            if self.dates["first_initial_qc_start_date"] is not None:
-                self.dates["first_initial_qc_start_date"] = sample.get(
-                    "first_initial_qc_start_date"
-                )
-
             log.info(f"Processing sample {sample_id}")
+
             # Check if the sample is aborted before processing
-            if sample.get("details", {}).get("status_(manual)") == "Aborted":
+            if sample_info.get("details", {}).get("status_(manual)") == "Aborted":
                 log.info(f"Sample {sample_id} is aborted, so skipping it")
-                self.aborted_samples[sample_id] = AbortedSampleInfo(
```

Comment:
Why not get rid of the `AbortedSampleInfo` class too at the same time?

Reply:
Whoops, missed it 😅